src/Server.js

```diff
@@ -817,7 +822,8 @@ function processSingleServerGrowth(server, numCycles) {
     // if there was any growth at all, increase security
     if(oldMoneyAvailable !== server.moneyAvailable) {
         //Growing increases server security twice as much as hacking
-        const usedCycles = numCycleForGrowth(server, server.moneyAvailable / oldMoneyAvailable);
+        let usedCycles = numCycleForGrowth(server, server.moneyAvailable / oldMoneyAvailable);
```

Comment:
*technically* can do these still as one line and keep the `const`...

Reply:
Yeah but it's easier to read in 2 lines